assets/src/scss/base/_colors.scss

```diff
@@ -1,19 +1,19 @@
-$white: #fff;
+$white: var(--white);
 $black: #000;
 
 // Grey
 $grey-80: #020202;
 $grey: #23292d;
-$grey-60: #45494c;
-$grey-40: #6f7376;
-$grey-20: #a3a5a7;
-$grey-10: #ececec;
-$grey-05: #f5f7f8;
+$grey-60: var(--grey-800);
```

Comment:
For now we can keep this variables, but I think we would need to remove all `$grey-*` afterwards and use directly the token var `--grey-*` but it's a `FLAGGED` implementation. 

The same with the rest of variables.